docs/development_suite/deploy/gitops-based/index.md

```diff
@@ -34,6 +34,16 @@ Another benefit is that **the cluster does not need to be accessed directly**, r
 
 Moreover, in push-based deployment models, pipelines often require access to credentials to authenticate with the cluster and make changes. Storing credentials within pipelines increases the risk of inadvertent exposure or misuse. In GitOps, **credentials are typically configured within the GitOps tool** which often resides on the cluster itself and has specific authorization rules that are not shared with the pipeline code and context.
 
+## Export and review configuration files
+
+When using **Enhanced Project Workflow**, Kubernetes files are generated on deployment. After selecting the desired revision, you will have the possibility to export and download the files in `tag.gz` format, or to review them directly inside the browser.
```

Comment:
It is possible to select also the version, not only the revision. Is it correct?

Reply:
Yes, how should I refer to it?

Reply:
We use somewhere the `reference` keyword?
We could:

1. After selecting the desired reference (revision or version) ...
2. After selecting the desired revision (or version), you will ...

What do you think about?

Reply:
I prefer the second option, I don't find other `reference` usages with this meanings and it sounds to me like a dev term